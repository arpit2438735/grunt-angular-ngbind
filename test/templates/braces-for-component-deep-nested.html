<some-tag>
    {{::someValue}}
    <div class="class"></div>
    <div>{{anotherTagValue}}</div>
</some-tag>