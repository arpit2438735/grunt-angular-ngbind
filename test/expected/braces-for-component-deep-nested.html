<some-tag><span ng-bind="::someValue"></span><div class="class"></div><div ng-bind="anotherTagValue"></div></some-tag>